libethereum/WarpCapability.h

```diff
@@ -117,6 +104,8 @@ class WarpCapability : public p2p::CapabilityFace
     void disablePeer(NodeID const& _peerID, std::string const& _problem);
 
 private:
+    static constexpr std::chrono::milliseconds s_backgroundWorkInterval{1000};
```

Comment:
Constants have `c_` prefix

Reply:
I saw some cases where static overrode const e.g.:

https://github.com/ethereum/aleth/blob/ae042a32103e4c421f942952bd42afdbd1e6b96f/libethereum/EthereumCapability.h#L131

I'll switch back to using c_ (and update the above instance)